src/user_actions_handler.py

```diff
@@ -60,6 +61,62 @@ def handler_sort(dir_path):
 
     return "Done!"
 
+    
+@input_error
+def add_note_handler(data):
+    
+     tags = []
+     new_data = data
+     for i in data:
+         if '#' in i:
+             
+             tags.append(i)
+             
+             
+     for i in new_data:
+         for j in tags:
+             if j == i:
+                 new_data.remove(i)     
+
+    
+     if tags:
+         print(tags)
+         for i in new_data:
+            for j in tags:
+                if j == i:
+                    new_data.remove(i)
+         text = ' '.join(new_data)
+     else:
+         text = data
+
+
+
+     if tags:
+        n = note_book.Note(text, tags)
+        note = note_book.NoteBook()
+
+        return print(note.add_note(n))
+     
+     else:
+        n = note_book.Note(text)
+        note = note_book.NoteBook()
+
+        return print(note.add_note(n))
+
+@input_error    
+def edit_note_handler(data):
+    
+    n = note_book.Note(data[0])
+    return print(n.edit_note(data[1]))
+
+ 
+def sort_note_handler():
```

Comment:
```suggestion
def sort_note_handler(*args):
```

arguments here are required to be added.